Makefile

```diff
@@ -0,0 +1,70 @@
+# copied from thirdparty/make-many-rubies/Makefile
```

Comment:
I suppose I should add our own license header here since I did modify this copy of the Makefile to work for our needs

Reply:
yes

Reply:
well, I think so.  Check with Roland.